pre_push.py

```diff
@@ -52,7 +52,7 @@ def run_static():
             path.join(current_directory, "tools", "check_documentation.py"),
         ]
     )
-    success &= do_process(["black ."], shell=True)
+    success &= do_process(["black", "."], shell=True)
```

Comment:
Theoretically, I assume this means we can get rid of the `shell=True`, since we're no longer relying on the shell to provide arguments for us. 
```suggestion
    success &= do_process(["black", "."])
```

Reply:
> Theoretically, I assume this means we can get rid of the `shell=True`, since we're no longer relying on the shell to provide arguments for us.

Yes, I think so. I didn't even consider this.